app/assets/locales/messages_ccodk_default.txt

```diff
@@ -406,6 +406,13 @@ template.malformed.chevron=The template file set for your print job has improper
 cannot.set.template=Could not set your print template
 no.file.browser=Your device does not have a file browser installed. Please download one from the playstore and then try again.
 
+empty.fields=Please complete all fields
+low.year=Year must be > 1900
+invalid.month=Month is invalid
+high.date=Date can be at most
+low.date=Date must be at least 1
+high.year=Year can be at most

```

Comment:
Can you prefix all these keys with something like `calendar` or `date`? Something to give users who want to override them for different languages some context